sat/cli/status/status_module.py

```diff
@@ -470,44 +471,32 @@ def can_use():
         return (True, None)
 
     def get_image_for_component(self, raw_component):
-        """Helper function to query the image name given a component dict from BOS.
+        """Helper function to query the image ID given a component dict from BOS.
 
         Args:
             raw_component (dict): a component dictionary returned from BOS v2
 
         Returns:
-            str: the name of the IMS image currently booted on the given
+            str: the IMS image ID currently booted on the given
                 component, or MISSING_VALUE if it cannot be identified
         """
-
-        # To get a human readable name for each booted image, IMS is
+        # To get a unique identifier for each booted image, IMS is
         # queried for images based on the path to the actual booted kernel.
         # This depends on the fact that the IMS image ID is part of the
         # path to the booted kernel in S3.
```

Comment:
We're no longer querying IMS for this info, so I would suggest shortening this comment to just:

```
        # Get the IMS image ID from the path to the booted kernel in S3.
```